src/main/java/ch/njol/skript/expressions/ExprEntityStorageMaxEntities.java

```diff
@@ -0,0 +1,77 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.util.Math2;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.block.Block;
+import org.bukkit.block.EntityBlockStorage;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.function.Consumer;
+
+@Name("Entity Storage Max Entities")
+@Description("The max number of entities an entity block storage (i.e. beehive) can store.")
+@Examples("set the entity blockstorage max entities of {_beehive} to 100")
+@Since("INSERT VERSION")
+public class ExprEntityStorageMaxEntities extends SimplePropertyExpression<Block, Integer> {
+
+	static {
+		registerDefault(ExprEntityStorageMaxEntities.class, Integer.class, "[entity block storage] max entities", "blocks");
```

Comment:
can't this just be part of the previous expression as a tag (also make sure to allow maximum)